v2/solver.py

```diff
@@ -63,6 +63,10 @@ def importUI(self, ui):
         for row in data:
             try:
                 name = row[0].lower().replace(' ','_')
```

Comment:
Yet again, this is not the proper place to set defaults. Please drop this commit it has already been implemented through the global vars and in the Ingredient class!